packages/ui/src/InteractiveListFormik.tsx

```diff
@@ -29,15 +36,18 @@ export const InteractiveListFormik = <V extends object>({ name, validate, ...pro
       name={name}
       render={(arrayHelpers) => {
         return (
-          <InteractiveList<V>
+          <InteractiveList
+            ref={inputRef}
```

Comment:
I would suggest to use a custom prop, say, `inputStateControlsRef` instead of a forwarded ref as forwarded refs are usually point to the dom elements. WDYT?

Reply:
You're totally right!